action.yml

```diff
@@ -15,6 +15,9 @@ inputs:
       Target image(s) that will be published to the registries configured in the file ${HOME}/.docker/config.json
       Type: CSV
     required: true
+  tar-file:
```

Comment:
if this is a path maybe we should call the input as `tar-path` exactly how [kaniko](https://github.com/GoogleContainerTools/kaniko?tab=readme-ov-file#flag---tar-path) specifies it.